packages/terra-tabs/src/common-tabs/_HiddenTab.jsx

```diff
@@ -129,20 +150,42 @@ const HiddenTab = ({
     { 'is-disabled': isDisabled },
     theme.className,
   );
+  const tabDeleteLabel = intl.formatMessage({ id: 'Terra.tabs.hint.removable' });
 
   const handleOnSelect = (event) => {
     event.preventDefault();
     event.stopPropagation();
 
     enableFocusStyles(event);
     onSelect(itemKey, metaData);
-    onChange(event, itemKey);
+    if (onChange) {
+      onChange(event, itemKey);
+    }
   };
 
+  function onCloseClick(event) {
+    event.stopPropagation();
+    onClosingTab(itemKey, metaData, event);
```

Comment:
require check for disabled tabs prior to onClose callback invocation.

Reply:
Done https://github.com/cerner/terra-framework/commit/628052153b88461ef88733be6b05c1fb519e3f41